keyboards/the_uni/keymaps/default/keymap.c

```diff
@@ -36,41 +36,26 @@ enum uni_keycodes {
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 /*
 	 [_QWERTY] = LAYOUT(
-	 	KC_Q, KC_W, KC_E, KC_R, LT(_CHOOSE, KC_T), KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, 
-	 	KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, 
+	 	KC_Q, KC_W, KC_E, KC_R, LT(_CHOOSE, KC_T), KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC,
+	 	KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT,
 	 	            KC_1, KC_C, KC_V, KC_N, KC_M, KC_2),
 
 	 [_CHOOSE] = LAYOUT(
-	 	_______, _______, _______, _______, _______, PLOVER, _______, _______, _______, _______, _______, 
-	 	_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, 
+	 	_______, _______, _______, _______, _______, PLOVER, _______, _______, _______, _______, _______,
+	 	_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
 	 	            _______, _______, _______, _______, _______, _______),
 */
 	[_PLOVER] = LAYOUT(
-		STN_S1,  STN_TL,  STN_PL,  STN_HL,  STN_ST1,  STN_ST3,  STN_FR,  STN_PR,  STN_LR,  STN_TR,  STN_DR , 
-		STN_S2,  STN_KL,  STN_WL,  STN_RL,  STN_ST2,  STN_ST4,  STN_RR,  STN_BR,  STN_GR,  STN_SR,  STN_ZR , 
+		STN_S1,  STN_TL,  STN_PL,  STN_HL,  STN_ST1,  STN_ST3,  STN_FR,  STN_PR,  STN_LR,  STN_TR,  STN_DR ,
+		STN_S2,  STN_KL,  STN_WL,  STN_RL,  STN_ST2,  STN_ST4,  STN_RR,  STN_BR,  STN_GR,  STN_SR,  STN_ZR ,
 		                  STN_N1,  STN_A,   STN_O,    STN_E,    STN_U,   STN_N2),
 
 };
 
-/*
-switch (keycode) {
  bool process_record_user(uint16_t keycode, keyrecord_t *record) {
- 	case PLOVER:
-       if (!record->event.pressed) {
-		 layer_on(_PLOVER);
-       }
-       return false;
-       break;
- 	case QWERTY:
-       if (!record->event.pressed) {
-		 layer_on(_QWERTY);
-       }
-       return false;
-       break;
- 	}
- 	return true;
+     set_single_persistent_default_layer(_PLOVER);
+     return true;
  }
```

Comment:
This should be removed, as running `set_single_persistent_default_layer` every call of `process_record_user` is not valid